src/main/webapp/jsp/instructorStudentListAjax.jsp

```diff
@@ -1,189 +1,3 @@
-                                    <ul class="dropdown-menu align-left" role="menu" aria-labelledby="dLabel">
-                                        <li role="presentation">
-                                            <a target="_blank"
-                                               role="menuitem"
-                                               tabindex="-1"
-                                               href="<%= data.getCourseStudentDetailsLink(student, data.account.googleId).replace("%40", "@") %>&addComment=student">
-                                                Comment on <%= data.sanitizeForHtml(student.name) %>
-                                            </a>
-                                        </li>
-                                        <li role="presentation">
-                                            <a target="_blank"
-                                               role="menuitem"
-                                               tabindex="-1"
-                                               href="<%= data.getCourseStudentDetailsLink(student, data.account.googleId).replace("%40", "@") %>&addComment=team">
-                                                Comment on <%= data.sanitizeForHtml(team.name) %>
-                                            </a>
-                                        </li>
-                                        <% if (data.hasSection) { %>
-                                            <li role="presentation">
-                                                <a target="_blank"
-                                                   role="menuitem"
-                                                   tabindex="-1"
-                                                   href="<%= data.getCourseStudentDetailsLink(student, data.account.googleId).replace("%40", "@") %>&addComment=section">
-                                                    Comment on <%= data.sanitizeForHtml(section.name) %>
-                                                </a>
-                                            </li>
-                                        <% } %>
-                                    </ul>
-                                </div>
-                            </td>
-                        </tr>
-                    <% }
-                }
-            } %>
-        </tbody>
-    </table>
-<% } else { %>
-    <table class="table table-responsive table-striped table-bordered margin-0">
-        <thead class="background-color-medium-gray text-color-gray font-weight-normal">
-            <tr>
-                <th class="align-center color_white bold">There are no students in this course</th>
-            </tr>
-        </thead>
-    </table>
-<% } %>
\ No newline at end of file
+<%@ taglib tagdir="/WEB-INF/tags/instructor/studentList" prefix="tisl" %>
+<tisl:ajaxResult courseId="${data.courseId}" courseIndex="${data.courseIndex}" hasSection="${data.hasSection}" sections="${data.sections}" />
```

Comment:
If the jsp file has no more than a single tag, would it be better to just place whatever is in the tag in it?
I don't see much purpose in having a separate tag with effectively the same result.